internal/pkg/cli/svc_init.go

```diff
@@ -91,6 +91,11 @@ These messages can be consumed by the Worker Service.`
 "Internet" will configure your service as public.`
 
 	wkldInitImagePrompt = fmt.Sprintf("What's the %s ([registry/]repository[:tag|@digest]) of the image to use?", color.Emphasize("location"))
+
+	fmtStaticSiteInitDirFilePrompt      = "Which " + color.Emphasize("files or directories") + " would you like to upload for %s?\nDirectories are recursively uploaded."
+	staticSiteInitDirFileHelpPrompt     = "Files or (recursive) directories to use for building your static site."
```

Comment:
I can be wrong, but I feel like "recursive" is already implied by "directories", unless it's valid/common to building a static site with an empty dir 🤔

Reply:
My main concern is not wanting users to think they have to select every subdir/file within the target dir. 🤷🏼‍♀️

Reply:
i think it's probably ok to remove because when you select a folder in a file selector, usually all files in that folder will be selected by default 🤔 We could adjust accordingly if it is confusing to some customers.

Reply:
But we don't do that in our multi select.... I started writing some code to make it do it, but then realized the right arrow selects all, which works great if the user filters by the dir name. But I'm not sure how to make this functionality more discoverable. 🤔

Reply:
I guess you're both right, that it's kind of common sense that it's recursive.